esx_service/cli/vmdkops_admin.py

```diff
@@ -1301,32 +1305,66 @@ def config_rm(args):
     # This asks for double confirmation, and removes the local link or DB (if any)
     # NEVER deletes the shared database - instead prints help
 
-    if not args.local:
+    if not args.local and not args.unlink:
         return err_out("""
-        Shared DB removal is not supported. For removing  local configuration, use --local flag.
-        For removing shared DB,  run 'vmdkops_admin config rm --local' on ESX hosts using this DB,
-        and manually remove the {} file from shared storage.
+        DB removal is irreversible operation. Please use '--local' flag for removing DB in SingleNode mode,
+        and use '--unlink' to unlink from DB in MultiNode mode.
+        Note that '--unlink' will not remove a shared DB, but simply configure the current ESXi host to stop using it.
+        For removing shared DB, run 'vmdkops_admin config rm --unlink' on ESXi hosts using this DB, and then manually
+        remove the actual DB file '{}' from shared storage.
         """.format(auth_data.CONFIG_DB_NAME))
 
+    if args.local and args.unlink:
+        return err_out("""
+        Cannot use '--local' and '--unlink' together. Please use '--local' flag for removing DB in SingleNode mode,
+        and use '--unlink' to unlink from DB in MultiNode mode.
+        """
+        )
+
     if not args.confirm:
         return err_out("Warning: For extra safety, removal operation requires '--confirm' flag.")
```

Comment:
Not this change, why does the user need to provide two options?? --local and --confirm, it would make sense if the confirmation is asked by the CLI. If the user provides "rm --local" then they want to remove the DB, the user typing out another "--confirm" doesn't make any difference - they still want to remove the DB.